govc/test/vm.bats

```diff
@@ -1036,7 +1036,7 @@ load test_helper
   run govc vm.customize -vm DC0_H0_VM0 -type Linux
   assert_failure # no -ip specified
 
-  run govc vm.customize -vm DC0_H0_VM0 -ip 10.0.0.45 -netmask 255.255.0.0 -type Linux
+  run govc vm.customize -vm DC0_H0_VM0 -ip 10.0.0.45 -netmask 255.255.0.0 -type Linux -dns-server 1.1.1.1 -dns-suffix example.com
```

Comment:
Can you provide a test which show/validates multiple (ie list)?

Reply:
In real vCenter these values would land inside the VM, files such as /etc/resolv.conf, /etc/hosts, etc.  The API (and hence vcsim) doesn't have access to them.